src/core_plugins/metrics/index.js

```diff
@@ -15,7 +15,9 @@ export default function (kibana) {
       return Joi.object({
         enabled: Joi.boolean().default(true),
         chartResolution: Joi.number().default(150),
-        minimumBucketSize: Joi.number().default(10)
+        minimumBucketSize: Joi.number().default(10),
+        tablePartitionSize: Joi.number().default(50),
```

Comment:
Why make these configuration options global? I think it would be must more flexible if it were on a per table basis instead.

Reply:
Let’s do both... I think admins should have a way to put safe guards on their systems.